.vib/minio-operator/runtime-parameters.yaml

```diff
@@ -0,0 +1,103 @@
+extraDeploy:
+# Taken from https://github.com/minio/operator/tree/master/examples
+- |
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: storage-configuration
+    namespace: {{ include "common.names.namespace" . | quote }}
+  stringData:
+    config.env: |-
+      export MINIO_ROOT_USER="minio"
+      export MINIO_ROOT_PASSWORD="minio123"
+      export MINIO_STORAGE_CLASS_STANDARD="EC:2"
+      export MINIO_BROWSER="on"
+      export NAMESPACE={{ include "common.names.namespace" . | quote }}
+  type: Opaque
+- |
+  apiVersion: v1
+  stringData:
+    CONSOLE_ACCESS_KEY: console
+    CONSOLE_SECRET_KEY: console123
+  kind: Secret
+  metadata:
+    name: storage-user
+    namespace: {{ include "common.names.namespace" . | quote }}
+  type: Opaque
+- |
+  apiVersion: minio.min.io/v2
+  kind: Tenant
+  metadata:
+    annotations:
+      prometheus.io/path: /minio/v2/metrics/cluster
+      prometheus.io/port: "9000"
+      prometheus.io/scrape: "true"
```

Comment:
Is this required on this object?

Reply:
I don't believe so. I just copied it from the base example